Backend/dev/run.sh

```diff
@@ -22,6 +22,7 @@ provider-dashboard
 image-api-helper
 driver-offer-allocator
 kafka-consumers
+special-zone
```

Comment:
This file doesn't exist anymore; instead you want to add this entry to `run-mobility-stack.nix`